### PR TITLE
Add PKGBUILD

### PR DESCRIPTION
**Pull request by johnramsden**
Add a git `PKGBUILD`.

Usage:

```
makepkg -sic -p PKGBUILD-git
```

### CODE REVIEW

**Comment by Kharacternyk** [comment]
The PKGBUILD is nice. There are currently some obstacles for packaging though:

* We need to expose a configuration mechanism that isn't "just modify the source" because it doesn't work well with packaged files.
* The current README doesn't serve good as documentation.

**Comment by johnramsden** [comment]
I added some command-line arguments so settings can be made via cli.

Example:

```
pacwall -h
USAGE: pacwall
        [ -b BACKGROUND ]
        [ -d NODE_HEX_COLOR ]
        [ -e EXPLICIT_NODE_COLOR ]
        [ -s EDGE_HEX_COLOR ]
        [ -g GSIZE ]
        [ -o OUTPUT ]
```

```
pacwall -b darkslategray -d '#dc143fff' -e darkorange -s '#ffff4444' -o p.png
```

File: pacwall.sh [hunk 1]
```diff
@@ -106,10 +115,43 @@ main() {
...
 }
 
+help() {
+    printf "%s\n\t%s\n\t%s\n\t%s\n\t%s\n\t%s\n\t%s\n" \
+        "USAGE: $0" \
+        "[ -b BACKGROUND ]" \
+        "[ -d NODE_HEX_COLOR ]" \
+        "[ -e EXPLICIT_NODE_COLOR ]" \
+        "[ -s EDGE_HEX_COLOR ]" \
+        "[ -g GSIZE ]" \
```
Comment (Kharacternyk) [inline pacwall.sh]: The colors of nodes and edges can also be specified using [X11 color names](https://en.wikipedia.org/wiki/X11_color_names), just like BACKGROUND and EXPLICIT_NODE_COLOR can also be hex values. All these colors are of the same format.

File: pacwall.sh [hunk 2]
```diff
@@ -60,31 +60,40 @@ compile_graph() {
...
+
+    # Optional arguments
+    if [ -n "${GSIZE}" ]; then
+        twopi_args+=("-Gsize=${GSIZE}")
+    fi
+    
+    twopi "${twopi_args[@]}" > "${OUTPUT}"
 }
 
 try_set_wallpaper() {
```
Comment (Kharacternyk) [inline pacwall.sh]: This function is more like `resize_wallpaper` now.

File: README.rst [hunk 3]
```diff
@@ -14,28 +14,16 @@ Requirements
 Customization
 -------------
 
-Any customizations should be performed by modifying the script itself. The code in the script is well-structured (should be). To discover the customization possibilities, read the man page of ``graphviz`` and ``twopi``, particularly the section on *GRAPH, NODE AND EDGE ATTRIBUTES*.
+Customizations can be made on the commandline, see the options with the ``-h`` flag.
+
```
Comment (Kharacternyk) [inline README.rst]: Maybe it would be nice to duplicate the help message in the README?

Comment (johnramsden) [reply README.rst]: Good idea.

File: pacwall.sh (hunk 2 above)
Comment (johnramsden) [reply pacwall.sh]: Renamed.

File: pacwall.sh (hunk 1 above)
Comment (johnramsden) [reply pacwall.sh]: Removed `HEX` from help strings.

**Pull request merged**